tests/data.py

```diff
@@ -1,22 +1,90 @@
 from enum import Enum
+from typing import Iterable
+
+
+class UserAuthenticationLevel(Enum):
+    """
+    Enum capturing the different supported authentication assurance levels supported for user-restricted access.
+    """
+
+    AAL3 = "aal3"
+    AAL2 = "aal2"
+    AAL1 = "aal1"
 
 
 class Actor(Enum):
     # User ID, org code, business function, IAL, OBO User ID
-    RC = ("555032000100", "D82106", "REFERRING_CLINICIAN", "3")
```

Comment:
Is this comment correct for the order of the parameters?

Reply:
Nope well spotted, I'll update these to properly match the arguments.